PluginCode/src/main/java/smr/cs/ualberta/libcomp/action/ReplacementAction.java

```diff
@@ -409,17 +421,21 @@ public void update(@NotNull final AnActionEvent event) {
             FileType fileType = psiFile.getFileType();
             if (fileType.getDefaultExtension().equalsIgnoreCase("java")) {
                 try {
-                    detectImportOnAction(event);
+                    detectOnAction(event, "java");
```

Comment:
is it better to create an enumeration with the file types and pass that instead of actual strings that always need to be compared etc?

Reply:
updated!